apps/explorer/src/pages/transaction-result/TransactionData.tsx

```diff
@@ -150,123 +70,7 @@ export function TransactionData({ transaction }: Props) {
-                                    </DescriptionItem>
-                                    <DescriptionItem
-                                        title={
-                                            <Text variant="pBody/semibold">
-                                                Computation Fee
-                                            </Text>
-                                        }
-                                    >
-                                        <GasAmount
-                                            amount={Number(
-                                                gasUsed?.computationCost
-                                            )}
-                                        />
-                                    </DescriptionItem>
-
-                                    <DescriptionItem
-                                        title={
-                                            <Text variant="pBody/semibold">
-                                                Storage Fee
-                                            </Text>
-                                        }
-                                    >
-                                        <GasAmount
-                                            amount={Number(
-                                                gasUsed?.storageCost
-                                            )}
-                                        />
-                                    </DescriptionItem>
-
-                                    <div className="mt-2 flex flex-col gap-2 rounded-xl border border-success px-4 py-2 md:flex-row md:items-center md:gap-4">
-                                        <div className="w-full md:w-40">
-                                            <Text
-                                                variant="pBody/semibold"
-                                                color="success-dark"
-                                            >
-                                                Storage Rebate
-                                            </Text>
-                                        </div>
-
-                                        <div className="ml-0 min-w-0 flex-1 leading-none">
-                                            <GasAmount
-                                                isHighlighted
-                                                amount={
-                                                    -Number(
-                                                        gasUsed?.storageRebate
-                                                    )
-                                                }
-                                            />
-                                        </div>
-                                    </div>
-                                </div>
-                            </TransactionBlockCardSection>
-                        </TransactionBlockCard>
+                        <GasBreakdown transaction={transaction} />
```

Comment:
instead of using `transaction` here - using `summary` and adding any computations / derivations into the summary function will be a net positive and will allow us to centralize all of this one-off logic into one place